Gemfile

```diff
@@ -0,0 +1,35 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 5.0'
+# Use sqlite3 as the database for Active Record
```

Comment:
drop all comments that are not documentation. You all should use postgresql in all ENV  in whole project.